website/static/files/tutorials/deploy-clusterlink.md

```diff
@@ -0,0 +1,62 @@
+
+1. Create the fabric and peer certificates for the clusters:
+
+    *Client cluster*:
+
+    ```sh
+    clusterlink create fabric
+    clusterlink create peer-cert --name client
+    ```
+
+    *Server cluster*:
+
+    ```sh
+    clusterlink create peer-cert --name server
+    ```
+
+    For more details regarding fabric and peer see [core concepts][].
```

Comment:
this mandates that `[core concepts]` is defined by anyone who includes this file.
Is it possible (I haven't tries) to define it at the bottom of this file and remove the reliance on including files?